### PR TITLE
docs: Change community contributions in README

### PR DESCRIPTION
**Pull request by toschdev**
Community Contributions should be in their own repository, then linked to the `app-registry` repository where all Apps can be found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +64,38 @@ Each directory in the root of this repository is a Go module containing an Ignit
...
 For support, questions, or feedback, please open an issue in this repository.
 
+### **Community Build Apps**
+
+The Ignite Apps ecosystem thrives on community contributions, offering a space for developers to share, showcase, and collaborate on Ignite integrations. If you've built an app that leverages Ignite CLI and wish to share it with the community, we encourage you to do so by following these guidelines:
+
+### **Create Your Repository**
+
+1. **Setup Your Project:** Begin by creating a new repository for your project on GitHub. This repository will host your app's code, documentation, and other relevant materials.
+2. **Choose a License:** It's important to choose an appropriate license for your project. This ensures that others know how they can legally use, modify, and distribute your app. If you're unsure which license to use, GitHub's licensing guide can help you decide.
```
Comment (Pantani) [inline README.md:74]:
```suggestion
1. **Setup Your Project:** Begin by creating a new repository for your project on GitHub. This repository will host your app's code, documentation, and other relevant materials.
2. **Choose a License:** Choosing an appropriate license for your project is essential. This ensures that others know how they can legally use, modify, and distribute your app. GitHub's licensing guide can help you decide which license to use if you're unsure.
```

Comment (Pantani) [reply README.md:74]: just a suggestion

**Review by Pantani — APPROVED** [review]
LGTM

just one suggestion

**Review by jeronimoalbi — APPROVED** [review]

**Review by salmad3 — APPROVED** [review]
Nice:) Two fast follows:

- May we add a real example?
- Let's ensure we add links (like to the app registry repo, apps directory, etc.)

**Review by Pantani — APPROVED** [review]

**Pull request merged**